Gemfile

```diff
@@ -13,7 +13,8 @@ gem 'bootsnap', '>= 1.1.0', require: false
 gem 'haml-rails'
 
 group :development, :test do
-  gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
+  gem 'chromedriver-helper'
+  gem 'selenium-webdriver'
```

Comment:
Hmm... where are the other gems in this group?

Reply:
They are added in PR#7:
https://github.com/CraftAcademy/D-pressed/pull/7